src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const res = {};
+
+  for (const keys in robot) {
+    if (res.hasOwnProperty(robot[keys])) {
+      return null;
+    } else {
+      res[robot[keys]] = keys;
+    }
```

Comment:
else is redundant here
```suggestion
    res[robot[keys]] = keys;
```